docs/user-guide/oauth-oidc-auth.md

```diff
@@ -28,6 +28,37 @@ With Auth0 as your IDP, you will need to create an `Application` to handle authe
   ![](/images/Auth0_none.png)
 
 
+## Keycloak
+
+With Auth0 as your IDP, you will need to create a `Client` to handle authentication requests from Ambassador Pro. The below instructions are known to work for Keycloak 4.8.
```

Comment:
Do you mean with Keycloak as your IDP?

Reply:
That's what I get for copy-pasta. fixed.